src/sentry/conf/server.py

```diff
@@ -1484,6 +1484,10 @@ def create_partitioned_queues(name):
 # dropped due to rate limits.
 SENTRY_METRICS_INDEXER_DEBUG_LOG_SAMPLE_RATE = 0.01
 
+# CloudSpanner metrics DB
+SENTRY_METRICS_CLOUDSPANNER_DB = None
+SENTRY_METRICS_CLOUDSPANNER_INSTANCE = None
```

Comment:
I think we are likely to run multiple spanner indexers in parallel (for example one based on a multi region and one based on a single region instance). So we will likely need multiple configurations. 

I think we can use the `db_backend` introduced in https://github.com/getsentry/sentry/pull/37745 to pick the right one.